src/server/prisma/schema.prisma

```diff
@@ -28,14 +28,28 @@ model recipes {
   products          String
   description       String
   image_link        String?
+  creator_id        Int?
   favourite_recipes favourite_recipes[]
   categories        categories          @relation(fields: [category_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "category")
+  users             users?              @relation(fields: [creator_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "creator_id_fk")
 }
 
 model users {
-  id                Int                 @id(map: "user_id") @default(autoincrement())
-  username          String              @unique(map: "username")
-  email             String              @unique(map: "email")
-  password          String
-  favourite_recipes favourite_recipes[]
+  id                                      Int                 @id(map: "user_id") @default(autoincrement())
+  username                                String              @unique(map: "username")
+  email                                   String              @unique(map: "email")
+  password                                String
+  image_link                              String?
+  favourite_recipes                       favourite_recipes[]
+  friendships_friendships_user1_idTousers friendships[]       @relation("friendships_user1_idTousers")
```

Comment:
Could you elaborate, why we need two array for friends?

Reply:
> Could you elaborate, why we need two array for friends?

The table friendships stores connection between two users. It contains of pair of foreign keys user1_id and user2_id and helps to store information about friends. It's also better that to store it in strings of arrays, because this way you can easily sort, group sql results, and make more complex sql requests